src/select/Menu.js

```diff
@@ -86,47 +106,29 @@ class Menu extends Component {
 
   render() {
     const {
-      allOption,
-      allowSelectAll,
-      focusedOption,
-      getOptionLabel,
-      labelKey,
       menuRef,
-      onOptionClick,
-      onOptionFocus,
-      optionComponent,
-      optionRef,
-      optionRenderer,
       options,
-      staticOption,
-      value,
-      valueKey
+      staticOption
     } = this.props
 
-    const OptionComponent = optionComponent
-    const selected = staticOption && isSelected(staticOption, value, valueKey)
-    const isFocused = staticOption && staticOption === focusedOption
+    if (staticOption) {
+      const isStaticOptionRendered =
+        options.find(o => o.abbreviation === staticOption.abbreviation)
```

Comment:
`abbreviation` is a dynamic property. Is this intended for `o[valueKey]` instead?

Reply:
Sounds good, I wasn't sure about that. Updated 👍